### DISCUSSION TITLE
Debug input

### DISCUSSION THREAD

**Discussion by CybotDNA** [Sixze/ALS-Refactored#175] [General]
Hi, I'm looking for the input from e.g. shift +1. 
I found all the other inputs, but I don't know how to disable the debug inputs. 

I would be very happy about an answer :-) 
Thank you for the great plugin!

**Comment by ameaninglessname** [comment]


**Reply by catarrestor** [reply]
I'm looking for the same thing, please help.

**Comment by Alex-G** [comment]
![Screenshot_2022-11-06-10-41-55-941_com.github.android.jpg](https://user-images.githubusercontent.com/5215249/200163951-45b63a24-3cc5-48fb-806e-1a6f398b1fa0.jpg)
